packages/legend-studio/src/models/protocols/pure/v1/model/valueSpecification/application/V1_AppliedProperty.ts

```diff
@@ -18,7 +18,7 @@ import type { V1_ValueSpecificationVisitor } from '../../../model/valueSpecifica
 import { V1_ValueSpecification } from '../../../model/valueSpecification/V1_ValueSpecification';
 
 export class V1_AppliedProperty extends V1_ValueSpecification {
-  class!: string;
+  class?: string;
```

Comment:
Why? Could you at least mention in the PR summary why we do this? Maybe break the PR summary down into multiple points and explain what you do to keep a record.